events/python/send_kombu.py

```diff
@@ -0,0 +1,47 @@
+import time
+
+from kombu import (
+    Exchange, Queue, Producer
+)
+
+from setup_kombu import (
+    ARTICLE_EXCHANGE_NAME,
+    DASHBOARD_QUEUE_NAME,
+    DELIVERY_MODE_PERSISTENT,
+    get_connection
+)
+
+
+if __name__ == '__main__':
+
+    with get_connection() as connection:
+        # create an exchange of type `fanout`, will skip if already exists
+        # is durable by default
+        exchange = Exchange(name=ARTICLE_EXCHANGE_NAME, type='fanout')
+
+        # create queue, will skip if exists
+        # is durable by default
+        # bind queue to exchange
+        queue = Queue(name=DASHBOARD_QUEUE_NAME, exchange=exchange)
+
+        with connection.channel() as channel:
+            producer = Producer(channel)
+
+            while True:
+                msg = f'Test message: {time.time()}'
+                print(f'Publishing: {msg}')
+
+                producer.publish(
+                    body=msg,
+                    delivery_mode=DELIVERY_MODE_PERSISTENT,
+                    routing_key=DASHBOARD_QUEUE_NAME,
```

Comment:
afaik a routing key should be optional?

Reply:
You are right, added it as was following the docs, but have now removed it.